dev_guide/daemonsets.adoc

```diff
@@ -0,0 +1,108 @@
+[[dev-guide-daemonsets]]
+= Using Daemonsets
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Daemonsets are used to run a copy of a pod on specific or all nodes in an
+{product-title} environment upon node creation.
+
+Use daemonsets to create shared storage, run a logging pod on every node in
+your cluster, or deploy a monitoring agent on every node.
+
+For more information on daemonsets, see the link:http://kubernetes.io/docs/admin/daemons/[Kubernetes documentation].
+
+[[dev-guide-creating-daemonsets]]
+== Creating Daemonsets
+
+[IMPORTANT]
+====
+Before creating daemonsets, ensure you have been
+xref:../admin_guide/manage_authorization_policy.adoc#admin-guide-granting-users-daemonset-permissions[given
+the required role by your {product-title} administrator].
+====
+
+When creating Daemonsets, the `*nodeSelector*` field is used to indicate the
+nodes on which the Daemonset should deploy onto.
+
+. Define the daemonset yaml file:
```

Comment:
I am not even sure what the question is - perhaps I am just not aware of what is going into 3.3 that will remove the need for creating a daemonset file.
@sferich888 can you please clarify?